.cirrus.yml

```diff
@@ -311,30 +293,25 @@ special_testing_task:
         - "vendor"
         - "build_each_commit"
 
-    gce_instance:
-        image_project: "libpod-218412"
-        zone: "us-central1-a"  # Required by Cirrus for the time being
-        cpu: 2
-        memory: "4Gb"
-        disk: 200
-        # A matrix could be used here, for now just one VM
-        image_name: "${FEDORA_CACHE_IMAGE_NAME}"
+    only_if: $CIRRUS_CHANGE_MESSAGE !=~ '.*\*\*\*\s*CIRRUS:\s*TEST\s*IMAGES\s*\*\*\*.*'
 
     env:
         matrix:
             SPECIALMODE: 'rootless'  # See docs
             SPECIALMODE: 'in_podman'  # See docs
 
-    timeout_in: 120m
+    timeout_in: 60m
 
     setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
     integration_test_script: '$SCRIPT_BASE/integration_test.sh |& ${TIMESTAMP}'
+    df_script: 'df -lhTx tmpfsyy'
```

Comment:
Is this `yy` a debugging leftover? A typo? Intentional? If debugging or typo, please remove or fix; if intentional, please add a comment describing its purpose. (Also line 314).

I'm really sorry. I desperately wanted to get this approved and merged because it is unmanageably huge, but this is too weird to allow.

Reply:
On second thought, can you just promise to fix this immediately once the PR merges? If so let's just lgtm this and get it merged.

Reply:
Gaah, no it's a typo.  Good catch, I found and thought I fixed that, apparently not.  I spent quite a number of hours debugging a disk-space problem related to building images, that would have been prevented by having the `df` output easily available.

Probably better fix and re-test everything.    This PR stands to affect so many people, I don't think it's possible to over-test it...

Reply:
Leaving it as-is does no harm to the run: it simply `df`s everything. The only problem will be if the code stays that way for a future maintenance programmer, it will cause wasted time. Merging right now, then submitting a fast-track PR to fix it, is IMO the sanest fix.